.github/pull_request_template.md

```diff
@@ -1,42 +1,11 @@
-<!--
-Thanks for sending a pull request!  Please keep the following tips in mind:
-  - Start the PR title with the related issue ID, e.g. '[AURON #XXXX] Short summary...'.
-  - Make your PR title clear and descriptive, summarizing what this PR changes.
-  - Provide a concise example to reproduce the issue, if possible.
-  - Keep the PR description up to date with all changes.
--->
-
 # Which issue does this PR close?
 
-<!--
-We generally require a GitHub issue to be filed for all bug fixes and enhancements and this helps us generate change logs for our releases. You can link an issue to this PR using the GitHub syntax. For example `Closes #123` indicates that this PR will close issue #123.
--->
-
-Closes #.
+Closes #<issue_number>
 
- # Rationale for this change
-<!--
- Why are you proposing this change? If this is already explained clearly in the issue then this section is not needed.
- Explaining clearly why changes are proposed helps reviewers understand your changes and offer better suggestions for fixes.  
--->
+# Rationale for this change
 
 # What changes are included in this PR?
-<!--
-There is no need to duplicate the description in the issue here but it is sometimes worth providing a summary of the individual changes in this PR.
--->
 
 # Are there any user-facing changes?
-<!--
-If there are user-facing changes then we may require documentation to be updated before approving the PR.
--->
-
-<!--
-If there are any breaking changes to public APIs, please add the `api change` label.
```

Comment:
This seems like its good to have, not very intuitive that we need to add the label w/o it.

Reply:
In fact we don't have such a label `api change`.